### PR TITLE
clightning: 0.12.0 -> 0.12.1

### PR DESCRIPTION
**Pull request by nepet**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Removed the prebuilt manpages and added lowdown for building `v0.12.1`

### CODE REVIEW

File: pkgs/applications/blockchains/clightning/default.nix [hunk 1]
```diff
@@ -7,6 +7,7 @@
 , automake
 , gettext
 , libtool
+, lowdown
```
Comment (prusnak) [inline pkgs/applications/blockchains/clightning/default.nix:10]: You are adding `lowdown` here, but not to the `buildInputs`

**Review by prusnak — CHANGES_REQUESTED** [review]
Please fix buildInputs and change commit message to read `clightning: 0.12.0 -> 0.12.1`

File: pkgs/applications/blockchains/clightning/default.nix (hunk 1 above)
Comment (nepet) [reply pkgs/applications/blockchains/clightning/default.nix:10]: Done! Just for me to understand: Why could I build the package locally without `lowdown` in the `buildInputs`?

Comment (prusnak) [reply pkgs/applications/blockchains/clightning/default.nix:10]: The upstream project ships local copy of lowdown which is used when system-wide installed one cannot be found - see https://github.com/ElementsProject/lightning/tree/master/external

Comment (nepet) [reply pkgs/applications/blockchains/clightning/default.nix:10]: Oh... Thanks!

**Comment by prusnak** [comment]
@GrahamcOfBorg build clightning

**Comment by prusnak** [comment]
One more thing, `lowdown` belongs to `nativeBuildInputs`, since it is needed only for build. Sorry about telling you earlier it belongs to `buildInputs`.

**Comment by prusnak** [comment]
@GrahamcOfBorg build clightning

**Pull request merged**